docs/migration_ov_2_0/snippets/ov_api/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(CMAKE_CXX_STANDARD 11)
+
+set(TARGET_NAME "ov_api_2_0_snippets")
+
+find_package(ngraph REQUIRED)
+find_package(InferenceEngine REQUIRED)
+
+file(GLOB SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
+
+add_library(${TARGET_NAME} STATIC ${SOURCES})
+
+target_link_libraries(${TARGET_NAME} PRIVATE inference_engine ngraph)
```

Comment:
do we need a special folder `docs/migration_ov_2_0`? We already have `docs/snippets`, can we put everything as `docs/snippets/ie_old_api.cpp` and ``docs/snippets/ov_new_api.cpp``?

Reply:
In general the idea was to use different cmake configs (OpenVINO vs IE + nGraph) in these scripts.